aspnetcore/security/ip-safelist.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Client IP safelist for ASP.NET Core
+author: damienbod
+description: Learn how to write Middleware or action filters to validate remote IP addresses against a list of approved IP addresses.
+ms.author: tdykstra
+ms.custom: mvc
+ms.date: 08/31/2018
+uid: security/ip-safelist
+---
+# Client IP safelist for ASP.NET Core
+
+By [Damien Bowden](https://twitter.com/damien_bod) and [Tom Dykstra](https://github.com/tdykstra)
+ 
+This article shows two ways to implement an IP safelist (also known as a whitelist):
```

Comment:
I'd move "By using ASP.NET Core" to this line. That makes the two bullets really stand out - middleware and filter